package.json

```diff
@@ -5,13 +5,17 @@
   "main": "index.js",
   "scripts": {
     "preinstall": "node ./bin/check-nodejs-version.js",
-    "test": "npm run check-types && npm run lint && npm run test-log-conventions && npm run test-unit && npm run test-misc",
+    "test": "npm run check && npm run test-log-conventions && npm run test-unit && npm run test-misc",
+    "check": "npm run check:types && npm run check:lint && npm run check:format",
```

Comment:
I moved the scripts up a bit, so their abstraction is closer to e.g. the `test` script. Like you would order functions of different level of abstractions in the code.